src/hooks/useVenueId.ts

```diff
@@ -19,5 +19,5 @@ export const useVenueId: () => string | null = () => {
     path: "/v/:venueId",
   });
 
-  return match?.params.venueId ? match.params.venueId : null;
+  return match?.params.venueId ? match.params.venueId : undefined;
```

Comment:
This would be the same but simpler yeah?

```suggestion
  return match?.params?.venueId;
```